main.go

```diff
@@ -177,28 +178,57 @@ func contains(slice []string, item string) bool {
 
 // main function to handle command-line flags and execute appropriate actions
 func main() {
-	listPtr := flag.Bool("list", false, "Print list of all pokemon")
-	namePtr := flag.String("name", "", "Select pokemon by name")
-	formPtr := flag.String("form", "", "Show an alternate form of a pokemon")
-	noTitlePtr := flag.Bool("no-title", false, "Do not display pokemon name")
-	shinyPtr := flag.Bool("shiny", false, "Show the shiny version of the pokemon instead")
-	randomPtr := flag.String("random", "1-8", "Show a random pokemon. This flag can optionally be followed by a generation number or range")
-	versionPtr := flag.Bool("version", false, "Show the cli version")
-	flag.Parse()
-
-	if *listPtr {
-		listPokemonNames()
-	} else if *versionPtr {
-		fmt.Println(version)
-	} else if *namePtr != "" {
-		showPokemonByName(*namePtr, !*noTitlePtr, *shinyPtr, *formPtr)
-	} else if *randomPtr != "" {
-		if *formPtr != "" {
-			fmt.Println("--form flag unexpected with --random")
-			os.Exit(1)
-		}
-		showRandomPokemon(*randomPtr, !*noTitlePtr, *shinyPtr)
-	} else {
-		flag.Usage()
+	app := &cli.App{
+		Flags: []cli.Flag{
```

Comment:
Name:  "pokego",
   Usage: "command-line tool that lets you display Pokémon sprites in color directly in your terminal.",

Reply:
needs to add the name and description.

Could you also add aliases for these as shown in image
![image](https://github.com/user-attachments/assets/5b7c413f-7254-4d40-8be5-623c50a9f1dd)

Reply:
https://cli.urfave.org/migrate-v1-to-v2/#flag-aliases-are-done-differently